pkg/config/data.go

```diff
@@ -118,15 +123,64 @@ type File struct {
 	Email            string `toml:"email"`
 	Endpoint         string `toml:"endpoint"`
 	LastVersionCheck string `toml:"last_version_check"`
+
+	// Allow the API endpoint itself to be dynamically switched if we wanted it to
+	APIEndpoint string `toml:"api_endpoint"`
```

Comment:
Not sure if this is useful, but could be interesting if at some point we wanted to dynamically change the backend serving up the configuration data.

Reply:
Isn't this what the `file.Endpoint` field on line 124 above is already for?

Reply:
oh really!? lol ok then 👍🏻 🙂